copy-to-hook.sh

```diff
@@ -0,0 +1,10 @@
+xcopy ./ibService ./../../.git/hooks/ibService/ /Y /E /F
```

Comment:
Привет.
Менял для себя
Делают все тоже под Linux - bash

2016-04-21 17:52 GMT+03:00 Shenja Sosna <notifications@github.com>:

> In copy-to-hook.sh
> <https://github.com/xDrivenDevelopment/precommit1c/pull/80#discussion_r60594438>
> :
>
> > @@ -0,0 +1,10 @@
> > +xcopy ./ibService ./../../.git/hooks/ibService/ /Y /E /F
>
> в linux нет xcopy, есть cp
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/xDrivenDevelopment/precommit1c/pull/80/files/aa1b1fb99f54486ae85a4b2c1366a4ea543d9a56#r60594438>
>